vbpd/vbpd-full/src/main/java/by/kirich1409/viewbindingdelegate/AnyLazyBindings.kt

```diff
@@ -0,0 +1,33 @@
+@file:Suppress("RedundantVisibilityModifier", "unused")
+@file:JvmName("ReflectionLazyAnyBindings")
+
+package by.kirich1409.viewbindingdelegate
+
+import android.view.View
+import androidx.recyclerview.widget.RecyclerView
+import androidx.viewbinding.ViewBinding
+import by.kirich1409.viewbindingdelegate.internal.ViewBindingCache
+
+/**
+ * Create new [ViewBinding] associated with the [Any]
+ *
+ * @param T Class of expected [ViewBinding] result class
+ * @param viewProvider Provide a [View] from the [Any].
+ */
+@JvmName("lazyViewBindingAny")
+public inline fun <reified T : ViewBinding> Any.lazyViewBinding(crossinline viewProvider: (Any) -> View) =
```

Comment:
This isn't good to use different names to create the delegate.  You need to use `viewBinding` everywhere

Reply:
if use same name
it conflicts with other viewBindings
this just makes less code to create viewBinding lazy
if you use only fragment or viewHolders but sometimes I try some experiment libraries for presentation layer
I'm using this personal and just wanted you to merge it to more developers can use

Reply:
Better name will be ```viewBindingLazy(...)```

Reply:
I've changed it